### PR TITLE
Use localized strings for file and folder status

### LINKED ISSUE #3598: Localize status strings in Git extension
### Dev Home version

_No response_

### Windows build number

_No response_

### Other software

_No response_

### Steps to reproduce the bug

Enable File Explorer integration and add a repository.
Browse to that repository in File Explorer.
Observe the folder status and the strings shown for modified/untracked files.

### Expected result

_No response_

### Actual result

Status strings such as "Modified", "Untracked", "Branch" are not being localized.

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by DefaultRyan**
## Summary of the pull request
The status strings we display show things like "Branch: ", "Modified", "Staged", etc. We need to localize these strings (and not localize things like git commit messages and such)

## Validation steps performed
Ran private build against repositories and ran unit tests

## PR checklist
- [ ] Closes #3598 
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw [hunk 1]
```diff
@@ -121,4 +121,44 @@
...
+    <value>Staged, Renamed, Modified</value>
+    <comment>File status to display when its Git status is renamed, and has both staged and unstaged changes</comment>
+  </data>
+  <data name="FileStatusModified" xml:space="preserve">
+    <value>Modified</value>
+    <comment>File status to display when its Git status has unstaged changes</comment>
+  </data>
+  <data name="FileStatusRenamedModified" xml:space="preserve">
+    <value>Renamed, Modified</value>
+    <comment>File status to display when its Git status is renamed with unstanged changes</comment>
```
Comment (ssparach) [inline extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw:162]:
```suggestion
    <comment>File status to display when its Git status is renamed with unstaged changes</comment>

```

File: extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw [hunk 2]
```diff
@@ -121,4 +121,45 @@
...
+    <comment>File status to display when its Git status is renamed, and has both staged and unstaged changes</comment>
+  </data>
+  <data name="FileStatusModified" xml:space="preserve">
+    <value>Modified</value>
+    <comment>File status to display when its Git status has unstaged changes</comment>
+  </data>
+  <data name="FileStatusRenamedModified" xml:space="preserve">
+    <value>Renamed, Modified</value>
+    <comment>File status to display when its Git status is renamed with unstaged changes</comment>
+
```
[the remark this answers is not in the document]
Comment (DefaultRyan) [reply extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw]: Fixed

**Review by ssparach — APPROVED** [review]

File: extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw [hunk 3]
```diff
@@ -121,4 +121,44 @@
     <value>Unable to get repository at the given root path</value>
     <comment>Error text for when the git extension is unable to obtain the repository given its root path</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="FolderStatusBranch" xml:space="preserve">
+    <value>Branch: {0}</value>
```
Comment (bbonaby) [inline extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw:125]: Do we still need to use the {Locked="{0}"} syntax in the comments?

e.g:

`<comment>{Locked="{0}"} Display the current Git branch</comment>`

Comment (krschau) [reply extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw:125]: I tried to find some documentation on this last week, and came up short. I did notice that Terminal doesn't do it.
https://github.com/microsoft/terminal/blob/17a55da0f9889aafd84df024299982e7b94f5482/src/cascadia/TerminalControl/Resources/es-ES/Resources.resw#L190

Comment (DefaultRyan) [reply extensions/GitExtension/FileExplorerGitIntegration/Strings/en-us/Resources.resw:125]: I'm not familiar with this Locked syntax. Got a link?

**Pull request merged**